### ISSUE TITLE
Question about hyper-parameters

### ISSUE THREAD

**Issue by CaptainRui1000** [bytedance/Protenix#124]
Hi, I trying to reproduce this project. However, there is no demo relative to your experiments. So I am not sure the hyper-parameters I used is correct or not.

According to the paper, I used the following parameter for the three stage.

#### for initial training
`--max_steps 75000 \
--train_crop_size 384 \
--diffusion_batch_size 48 \
--train_confidence_only False \
--loss.weight.alpha_pae 0.0 \
--loss.weight.alpha_diffusion 4.0 \
--loss.weight.alpha_distogram 0.03 \
--loss.weight.alpha_bond 0.0 \
--loss.weight.smooth_lddt 1.0 \`

#### for finetune stage 1
`--max_steps 16000 \
    --train_crop_size 640 \
    --diffusion_batch_size 32 \
    --train_confidence_only False \
    --loss.weight.alpha_pae 0.0 \
    --loss.weight.alpha_diffusion 4.0 \
    --loss.weight.alpha_distogram 0.03 \
    --loss.weight.alpha_bond 1.0 \
    --loss.weight.smooth_lddt 0.0 \`

#### for finetune stage 2
`--max_steps 16000 \
    --train_crop_size 640 \
    --diffusion_batch_size 32 \
    --train_confidence_only True \
    --loss.weight.alpha_pae 1.0 \
    --loss.weight.alpha_diffusion 0.0 \
    --loss.weight.alpha_distogram 0.0 \
    --loss.weight.alpha_bond 1.0 \
    --loss.weight.smooth_lddt 0.0 \`

Are they reasonable? Want to receive some advice.

Thanks for sharing your great work.

**Comment by zhangyuxuann** [comment]
@CaptainRui1000 Yes. Your parameter settings are basically the same as ours. We just set max_steps as the same in different stages and reload the weights and steps for further training. For example,
1. for initial training, we set max steps 100k, and run about 75k
```shell
--max_steps 100000 \ 
--train_crop_size 384 \ 
--diffusion_batch_size 48 \ 
--train_confidence_only false \ 
--loss.weight.alpha_pae 0.0 \ 
--loss.weight.alpha_diffusion 4.0 \ 
--loss.weight.alpha_distogram 0.03 \ 
--loss.weight.alpha_bond 0.0 \ 
--loss.weight.smooth_lddt 1.0 \
--lr 0.001 \
```

2. in finetune stage 1, load_checkpoint_path and load_ema_checkpoint_path is the weight from  initial training.
```shell
--max_steps 100000 \ 
--train_crop_size 640 \ 
--diffusion_batch_size 32 \ 
--train_confidence_only false \ 
--loss.weight.alpha_pae 0.0 \ 
--loss.weight.alpha_diffusion 4.0 \ 
--loss.weight.alpha_distogram 0.03 \ 
--loss.weight.alpha_bond 1.0 \ 
--loss.weight.smooth_lddt 0.0 \
--load_strict true \
--load_params_only false \
--skip_load_step false \
--skip_load_optimizer false \
--skip_load_scheduler true \
--load_checkpoint_path $load_checkpoint_path \
--load_ema_checkpoint_path $load_ema_checkpoint_path \
--lr 0.0005 \
```

**Comment by CaptainRui1000** [comment]
@zhangyuxuann Thanks for your answer.

And what about the fine-tuning stage 2 ?

In the paper, the **PAE Head** is turned on, and the **Structure & Distogram** is turned off. So I used the following hyper-parameters. 
`
--train_confidence_only True
--loss.weight.alpha_pae 1.0 
--loss.weight.alpha_diffusion 0.0 
--loss.weight.alpha_distogram 0.0
`

In the other hand, the **Diffusion Batch Size** and **Bond Loss Weight** were not specified. Does it mean their default values can work fine? Because, in the [config_base.py](https://github.com/bytedance/Protenix/blob/main/configs/configs_base.py),  it say **alpha_bond** and **smooth_lddt** should be changed for all fine-tuning stages, which makes me feel a little confuse. At last, I employed these below and ignore the **Diffusion Batch Size**.
 `
--loss.weight.alpha_bond 1.0
--loss.weight.smooth_lddt 0.0
`

Did I choose correctly?

**Comment by zhangyuxuann** [comment]
@CaptainRui1000 Yes, for fine-tuning stage 2, your setting is okay. If you set `--loss.weight.alpha_diffusion 0.0`, then Diffusion Batch Size, loss.weight.alpha_bond and loss.weight.smooth_lddt can be not specified. these parameters will not take effect if only training confidence head

![Image](https://github.com/user-attachments/assets/63ec4c7c-20fe-4d97-a38f-b8f7ec9ed48c)
